File: WebApp/src/main/webapp/app/model/dictionary/RoleModel.js
Ext.define('TransDocs.model.dictionary.RoleModel', {
    extend: 'TransDocs.model.AbstractModel',
    fields:[
        {name: 'roleName', type: 'string'},
        {name: 'description', type: 'string'}
    ],

    getObjectType: function(){
        return "td_role";
    }
});

File: Model/src/main/java/com/td/model/entity/dictionary/role/RoleModel_.java
package com.td.model.entity.dictionary.role;

import com.td.model.entity.AbstractModel_;

import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;

/**
 * Created by konstantinchipunov on 25.07.14.
 */
@StaticMetamodel(RoleModel.class)
public abstract class RoleModel_ extends AbstractModel_ {

    public static volatile SingularAttribute<RoleModel, String> roleName;

    public static volatile SingularAttribute<RoleModel, String> description;


}
